`limesctl project show` crashes when given a domain ID

To reproduce, take any domain ID, and run:

```
$ limesctl project show $DOMAIN_ID
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x7b9e36]

goroutine 1 [running]:
github.com/sapcc/limesctl/internal/core.FindProject(0xc000255810, 0xc000255860, 0x7fff2986b4e9, 0x20, 0x0, 0x0, 0x0, 0x0, 0x7f5bafe9a108, 0xc0000102a8, ...)
        /x/src/github.com/sapcc/limesctl/internal/core/find.go:179 +0xb16
main.main()
        /x/src/github.com/sapcc/limesctl/main.go:240 +0xda5
```

After getting the project, the code should inspect the `is_domain` attribute and bail out in an orderly fashion if the project is in fact a domain.

--- a/internal/core/find.go
+++ b/internal/core/find.go
@@ -173,7 +173,7 @@ func FindProject(identityV3, limesV1 *gophercloud.ServiceClient, userInputProjec
 	p, err := gopherprojects.Get(identityV3, userInputProject).Extract()
 	if err == nil {
 		if p.IsDomain {
-			return nil, errors.New("the given ID belongs to a domain, usage instructions: limectl domain --help")
+			return nil, errors.New("the given ID belongs to a domain, usage instructions: limesctl domain --help")
 		}
 
 		// get domain name

--- a/internal/core/find.go
+++ b/internal/core/find.go
@@ -172,6 +172,10 @@ func FindProject(identityV3, limesV1 *gophercloud.ServiceClient, userInputProjec
 	// Strategy 4: assume that userInputProject is an ID
 	p, err := gopherprojects.Get(identityV3, userInputProject).Extract()
 	if err == nil {
+		if p.IsDomain {
+			return nil, errors.New("the given ID belongs to a domain, usage instructions: limectl domain --help")
+		}
+
 		// get domain name
 		d, err := gopherdomains.Get(identityV3, p.DomainID).Extract()
 		var domainName string

